graphics/jpeg-quantsmooth/Portfile

```diff
@@ -35,10 +39,12 @@ post-patch {
     }
 }
 
-configure.cflags-append \
+if {[string match *clang* ${configure.compiler}]} {
```

Comment:
I thought `compiler.openmp_version` already handles picking compiler? Would these extra flags hurt anything?

Reply:
`libomp` is clang-specific library, gcc uses `libgomp` (and unlike clang, knows where it is).

Reply:
Got it. I'll wait until you and upstream has an agreement and a finalized patch.